client/components/pie-chart/legend-placeholder.js

```diff
@@ -5,43 +5,47 @@
  */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { range } from 'lodash';
 
-const SVG_SIZE = 30;
+/**
+ * Internal dependencies
+ */
+import LegendItemPlaceholder from 'components/legend-item/placeholder';
 
 class PieChartLegendPlaceholder extends Component {
 	static propTypes = {
-		numLegendElements: PropTypes.number.isRequired,
+		dataSeriesInfo: PropTypes.arrayOf(
+			PropTypes.shape( {
+				description: PropTypes.string,
+				name: PropTypes.string.isRequired,
+			} )
+		).isRequired,
+	};
+
+	static getDerivedStateFromProps( props, state ) {
+		const longestName = props.dataSeriesInfo.reduce( ( pv, cv ) => {
```

Comment:
Can we have a more descriptive name here? I'm not sure what `pv` and `cv` stands for...